src/main/java/com/iota/iri/conf/SnapshotConfig.java

```diff
@@ -6,67 +6,93 @@
 public interface SnapshotConfig extends Config {
 
     /**
-     * @return {@value Descriptions#LOCAL_SNAPSHOTS_ENABLED}
+     * Default Value: {@value BaseIotaConfig.Defaults#LOCAL_SNAPSHOTS_ENABLED}
+     *
+     * @return {@value SnapshotConfig.Descriptions#LOCAL_SNAPSHOTS_ENABLED}
      */
     boolean getLocalSnapshotsEnabled();
 
     /**
-     * @return {@value Descriptions#LOCAL_SNAPSHOTS_PRUNING_ENABLED}
+     * Default Value: {@value BaseIotaConfig.Defaults#LOCAL_SNAPSHOTS_PRUNING_ENABLED}
+     *
+     * @return {@value SnapshotConfig.Descriptions#LOCAL_SNAPSHOTS_PRUNING_ENABLED}
      */
     boolean getLocalSnapshotsPruningEnabled();
 
     /**
-     * @return {@value Descriptions#LOCAL_SNAPSHOTS_PRUNING_DELAY}
+     * Default Value: {@value BaseIotaConfig.Defaults#LOCAL_SNAPSHOTS_PRUNING_DELAY}
+     *
+     * @return {@value SnapshotConfig.Descriptions#LOCAL_SNAPSHOTS_PRUNING_DELAY}
      */
     int getLocalSnapshotsPruningDelay();
 
     /**
-     * @return {@value Descriptions#LOCAL_SNAPSHOTS_INTERVAL_SYNCED}
+     * Default Value: {@value BaseIotaConfig.Defaults#LOCAL_SNAPSHOTS_INTERVAL_SYNCED}
+     *
+     * @return {@value SnapshotConfig.Descriptions#LOCAL_SNAPSHOTS_INTERVAL_SYNCED}
      */
     int getLocalSnapshotsIntervalSynced();
 
     /**
-     * @return {@value Descriptions#LOCAL_SNAPSHOTS_INTERVAL_UNSYNCED}
+     * Default Value: {@value BaseIotaConfig.Defaults#LOCAL_SNAPSHOTS_INTERVAL_UNSYNCED}
+     *
+     * @return {@value SnapshotConfig.Descriptions#LOCAL_SNAPSHOTS_INTERVAL_UNSYNCED}
      */
     int getLocalSnapshotsIntervalUnsynced();
 
     /**
-     * @return {@value Descriptions#LOCAL_SNAPSHOTS_DEPTH}
+     * Default Value: {@value BaseIotaConfig.Defaults#LOCAL_SNAPSHOTS_DEPTH}
+     *
+     * @return {@value SnapshotConfig.Descriptions#LOCAL_SNAPSHOTS_DEPTH}
      */
     int getLocalSnapshotsDepth();
 
     /**
-     * @return {@value Descriptions#SNAPSHOT_TIME}
+     * Default Value: {@value BaseIotaConfig.Defaults#GLOBAL_SNAPSHOT_TIME}
```

Comment:
the default value name does not match the description name.
`GLOBAL_SNAPSHOT_TIME` vs `SNAPSHOT_TIME`